internal/annotations/annotations.go

```diff
@@ -32,8 +32,9 @@ const (
 )
 
 const (
-	IngressClassKey        = "kubernetes.io/ingress.class"
-	KnativeIngressClassKey = "networking.knative.dev/ingress.class"
+	IngressClassKey           = "kubernetes.io/ingress.class"
+	KnativeIngressClassKey    = "networking.knative.dev/ingress-class"
+	KnativeIngressClassAltKey = "networking.knative.dev/ingress.class"
```

Comment:
I'm open to suggestions whether this should follow [knative's naming convention](https://github.com/knative/networking/blob/894675d14dfd448b1326bd21799098d5c29f7b71/pkg/apis/networking/register.go#L82-L102) (which it doesn't do now) i.e. the `-` annotation is the "alt" where the original with `.` keeps its name.

Reply:
Done